data/json/items/armor.json

```diff
@@ -4430,6 +4430,28 @@
     "material_thickness": 4,
     "flags": [ "WATER_FRIENDLY" ]
   },
+  {
+    "id": "helmet_galea",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": "galea",
+    "name_plural": "galeae",
+    "description": "An Imperial Roman helmet, with distinct projections protecting the sides of the head.",
+    "weight": 1310,
+    "volume": 10,
+    "price": 35000,
+    "bashing": 10,
+    "material": [ "iron", "leather" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "covers": [ "HEAD" ],
+    "coverage": 85,
+    "encumbrance": 10,
+    "warmth": 10,
+    "material_thickness": 3,
+    "techniques": [ "WBLOCK_1" ],
```

Comment:
Do any other helmets add a blocking technique? This seems like a really weird thing for a helmet.

Reply:
Sure, the helmet below this one in the same file has the same technique. I figure it's only usable while the helmet is wielded as a weapon.

Reply:
Huh, yeah, it seems to be pretty common among armour. Odd.